src/Sylius/Bundle/AdminBundle/Resources/config/services.xml

```diff
@@ -44,6 +44,12 @@
         <service id="Sylius\Bundle\AdminBundle\Twig\OrderUnitTaxesExtension">
             <tag name="twig.extension"/>
         </service>
+
```

Comment:
```suggestion
```

Reply:
IMO, even though we are now trying to follow the convention when adding new stuff (and in this case separate al services with blank lines) but I think it looks like 💩 when we do it like that. It would look much better if we did the refactor first.